src/components/Kitchen/Kitchen.js

```diff
@@ -39,32 +42,23 @@ const Kitchen = () => {
           <Grid
             container
             justifyContent="space-evenly"
-            alignContent="center"
-            padding="10px"
+            alignItems="center"
           >
             {/*  1 of 2 items in CHILD container*/}
             {/* Recipe Display */}
-            <Grid item>
-              {/* Put this in for the first typography 
-                <RecipesDisplay  recipeList={kitchenRecipes}/> 
-            */}
+            <Grid item width="80%">
```

Comment:
I don't think `width` should ever be used on grid items since it defeats the purpose of using grid. Instead this should be `xs={9}` or something similar

```suggestion
            <Grid item xs={9}>
```